### ISSUE TITLE
ARIA support

### ISSUE THREAD

**Issue by bwente** [rewdy/leaveNotice#6]
The prompt needs to announce itself as a dialog.

`('<div role="alertdialog" title="Leaving Site Notice" id="' + options.messageHolderId + '"><div id="' + options.messageBoxId + '"></div></div>')`

**Comment by rewdy** [comment]
Good point. I probably should re-write the whole thing to make it more accessible... Let me see what I can do.

**Comment by bwente** [comment]
I am not sure how to do a pull request. But I have some code that I added. It is pretty simple. Can I send it to you?
